qlib/workflow/expm.py

```diff
@@ -15,23 +15,35 @@
 from ..log import get_module_logger
 from ..utils.exceptions import ExpAlreadyExistError
 
+
 logger = get_module_logger("workflow")
 
 
 class ExpManager:
     """
-    This is the `ExpManager` class for managing experiments. The API is designed similar to mlflow.
-    (The link: https://mlflow.org/docs/latest/python_api/mlflow.html)
+        This is the `ExpManager` class for managing experiments. The API is designed similar to mlflow.
```

Comment:
@ChiahungTai  You can start from these docs.

Reply:
"we can have multiple `Experiment`s with different uri" => IIUC, the words are imply the design of ExpManger is suggestion the user use different uri in each experiment? 

From my understanding in MLFLow, the uri is like a store backend(sql, file...). The ExpManger is a single entry to lookup and manipulate the experiments in the uri.
 
So the user can change different uri to retrieve the different topic of experiements.

Reply:
Yes.  
ExpManager is a singleton, but we can change its uri  to get experiments from different uri.
I added some comments just now.
Please check if it is helpful

Reply:
Your new comment is more clear than before.